src/pages/researcher.pug

```diff
@@ -0,0 +1,90 @@
+extends ../layouts/case-study.pug
+
+block prepend vars
+  -var pageTitle = "Researcher job - Leaf"
+  -var metaUrl = "https://weareleaf.com/researcher"
+  -var metaTitle = "Researcher"
+  -var metaDescription = "We're looking for a researcher to help drive change in our 4-year old our company, from how we target and win new business, to how we present ourselves to the World."
+
+block pageComponents
+  +banner({
+    bannerCompany: "leaf",
+    bannerHeading: "Researcher"
+  })
+
+  .section.section--sm-spacing
+    .container
+      +intro({
+        introImage: {
+          src: "/assets/images/researcher/logo.svg",
+          alt: "Leaf",
+        },
+        introText: "We're looking for a researcher to drive exciting changes at Leaf, helping us deliver on a more mature lead-generation strategy, and enhancing the value we can offer our clients."
+      })
+
+  .section
+    .container
+      figure
+        img(src="/assets/images/researcher/inline-1.webp" alt="Mike Carter" onerror="this.onerror=null; this.src='/assets/images/researcher/inline-1.jpg'")
+
+      .grid.grid--two-thirds-reverse
+        .grid__item
+          +summary({
+            summaryHeading: "Who are we?",
+            summaryText: "Leaf are a small product-focused studio in South-East England, helping organisations build better digital products."
+          })
+
+        .grid__item
+          h4 About the role
+          p As a company, we celebrate our 4<sup>th</sup> birthday in June, and while it has been a wonderful ride to be on, we're looking for a researcher to help ensure that the next 4 are every bit as fun.
+          p To date, Leaf's marketing strategy could be boiled down to something along the lines of, #[em 'people talk; do good work, and we'll win more']. Going forward, that strategy will be somewhat more calculated, and research is that heart of it. 
+
+      figure
+        img(src="/assets/images/researcher/inline-2.webp" alt="Chris and Mike" onerror="this.onerror=null; this.src='/assets/images/researcher/inline-2.jpg'")
+        figcaption Chris and Mike on a face-to-face team catch-up day
+
+      .grid.grid--two-thirds
+        .grid__item
+          h4 Primary responsibilities
+          ul.styled-list.styled-list--inside-lg
+            li.styled-list__item Devising and undertaking research to support Leaf's marketing strategy.
```

Comment:
I think we need to be more specific here, or elsewhere in this job description, what areas we'd want people do be doing research on. I think it's market research, and UX research primarily?